active/fellowship.md

```diff
@@ -2,30 +2,62 @@
+- Gathering and maintaining statistics on the impact and efficacy of the Fellow program (for use in fundraising, reports to sponsors, etc). See [reporting](#reporting) for a short discussion of this report.
+- TKTK am I missing anything?
 
 ## Initial membership
 
-- Chair: Brian Moloney
-- Board liaison: Chaim Kirby
+Upon adoption of this new charter, the Board will issue an open call for members, recruit, and select membership. We'll follow the guidelines outlined below under [future membership](#future-membership) in selecting this initial group./
+
+- Chair: TBD
+- Board liaison: TBD
+- Designated manager: TBD
 - Other members:
-  - Andrew Godwin
-  - Frank Wiles
+  - TBD
 
 ## Future membership
 
-Board-managed: the DSF Board appoints members of this WG.
+Board-managed: the DSF Board appoints members of this WG, including specific roles (chair, Fellow manager).
+
+The Board will select membership with some high-level goals in mind:
+
+- **Diversity**: the WG membership should be diverse, as representative of the global Django community as possible within the confines of a small team.
+- **Management experience**: WG members, _particularly_ the designated manager, should have some engineering management experience, or other relevant experience (i.e. experience with fellowship programs at other NPOs, etc). This isn't a hard
 
-Membership is open to volunteers, subject to board approval. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest.
+Membership is open to volunteers, subject to board approval. [Contact the board](https://www.djangoproject.com/contact/foundation/) to express interest. Please include a brief explanation of how your background aligns with the goals above.
+
+Members serve a 1-year term, April 1st to February 28/29th, in alignment with the Fellow contracts. At the end of this term, they need to opt into staying involved to keep being
+a member of the group. If any member wishes to leave the group before the end of their term, they can do so without a vote.
 
 ## Budget
 
-The DSF Board sets the Fellowship compensation, but the WG then has the power to select who fills the position.
+- The Board sets the Fellowship compensation (which is adjusted yearly, automatically, to match CPI inflation).
+- The Board establishes a yearly Travel & Expenses budget, which Fellows may spend against (with management approval). This is set to a level to allow Fellows to attend at least one DjangoCon.
+  - Expenses beyond this yearly budget may (probably will!) be approved, but would require approval by the board to increase the budget.
 
 ## Comms
 
-TBD
+The Fellowship-WG has a private mailing list, and uses the DSF Slack.
 
 ## Reporting
 
-TBD
+The Fellowship WG is required to submit two kinds of reports:
+
+- A report on the impact and efficacy of the Fellow program.
```

Comment:
Reflecting on the current Fellow reports, I find them useful but as-is wouldn’t know how to judge the efficacy of the program from them. I’m not sure what kind of information would be needed here, tough one.